pom.xml

```diff
@@ -99,7 +99,13 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
-            <version>2.4.0</version>
+            <version>2.4.0</version> <!-- should really be 2.6.4 to match scm-api but ok as git wass forward compat -->
+        </dependency>
+
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>scm-api</artifactId>
+            <version>2.0.3-SNAPSHOT</version>
```

Comment:
Because it adds useless dependency that participate in multibranch/branch api chaos with cycle dependencies and huge pages with instructions what to do with broken instances. I also have gh plugins depending on github-plugin and i don't want to deal with redundant dependencies. github-plugin as trigger and eventing mechanism may work without additional deps. Freely add Listener calls instead of deprecating it.  I would also like to see firstly such changes in gitlab and other trigger plugins, not only gh.